prompting/datasets/__init__.py

```diff
@@ -0,0 +1 @@
+from .wiki import WikiDataset, WikiDateDataset  # noqa: F401
```

Comment:
I suspect there's a reason for this but maybe not.  If we can remove this from the init.py, that'd be preferrable.